doc/life-cycles/cipher.dot

```diff
@@ -1,4 +1,6 @@
 digraph cipher {
+    bgcolor="#00000000";
```

Comment:
https://graphviz.org/doc/info/attrs.html#k:color

Try `bgcolor = "transparent"`?

Reply:
That's the same end result as setting the alpha channel to zero.  I guess being explicit is better.

Reply:
The output image files are identical with this change.